pkg/cmd/run/root.go

```diff
@@ -155,12 +155,11 @@ var runCmd = &cobra.Command{
 			// area.Clear()
 
 			stackState.UpdateFromWorkerEvent(we)
+
 			area.Update(
-				stackState.ApiTable(9001),
-				"\n\n",
-				stackState.TopicTable(9001),
-				"\n\n",
-				stackState.SchedulesTable(9001),
+				stackState.ApiTable(9001, ""),
+				stackState.TopicTable(9001, "\n\n"),
+				stackState.SchedulesTable(9001, "\n\n"),
```

Comment:
I see that you've added the newlines to conditionally print each of the tables and their prefixes if they are each empty.

Adding the newline characters to the method here makes it less clear what is actually happening. It isn't clear how it's being used or is going to be printed.

It might be better to externalise the printing logic and add the tables to a slices of strings instead, and finally print the joining of the strings.

e.g.
```go
tables := []string{}

if len(apis) > 0 {
  tables = append(tables, stackState.ApiTable(9001).
}

...other tables...

area.Update(strings.Join(tables, "\n\n"))
```
> Note I haven't checked any of the above for correctness, just a logic example.

Reply:
Re-written to exernalize '\n' formatting.

Reply:
you legend! looks awesome!